### PR TITLE
Events take pre-game lobby time into account

### PR DESCRIPTION
**Pull request by MrsTonedOne**
Meteors happening before the shuttle can even be called was lame.

### CODE REVIEW

File: code/modules/events/_event.dm [hunk 1]
```diff
@@ -40,7 +40,7 @@
 /datum/round_event_control/proc/canSpawnEvent(var/players_amt, var/gamemode)
 	if(occurrences >= max_occurrences)
 		return FALSE
-	if(earliest_start >= world.time)
+	if(earliest_start >= world.time-ticker.ound_start_time)
```
Comment (vuonojenmustaturska) [inline code/modules/events/_event.dm]: ound?

Comment (praisenarsie) [reply code/modules/events/_event.dm]: ound.

**Comment by CPTANT** [comment]
This will only change it 3 minutes or so, right?

File: code/modules/events/_event.dm [hunk 2]
```diff
@@ -40,7 +40,7 @@
 /datum/round_event_control/proc/canSpawnEvent(var/players_amt, var/gamemode)
 	if(occurrences >= max_occurrences)
 		return FALSE
-	if(earliest_start >= world.time)
+	if(earliest_start >= world.time-ticker.round_start_time)
```
Comment (JJRcop) [inline code/modules/events/_event.dm]: It's SSticker now.

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**